src/main/java/com/world/demo/worldmap/controller/CountryController.java

```diff
@@ -44,7 +39,7 @@ public ResponseEntity getCountryByCode(@PathVariable String code) {
             }
             return new ResponseEntity<>("INVALID_COUNTRY_CODE", HttpStatus.INTERNAL_SERVER_ERROR);
         } catch (Exception ex) {
-            Logger.getLogger(ApplicationController.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
You do not need to catch exceptions here like this, Spring will automatically map any RuntimeException to a 500: Internal Server Error anyway.